Getting Started.md

```diff
@@ -115,16 +104,11 @@ To disambiguate order of operations, use parentheses.
 
 Let's use some functions from the [`String` module](http://package.elm-lang.org/packages/elm-lang/core/latest/String).
 
-First, we need to import the module.
-
-
-```
-> import String
-```
-
-Then any function we want to use is namespaced under `String`:
+Any function we want to use is namespaced under `String`:
```

Comment:
`String` is unusual because it's one of the default imports.  Will this be confusing, or is `import` explained enough later when its needed?  Is it worth mentioning here the normally you'd need to `import` the module, but `String` is imported by default as one of the default imports?